store/router.go

```diff
@@ -87,8 +99,21 @@ func (r *Router) PutWithoutKey(ctx context.Context, value []byte) (key []byte, e
 	}
 
 	if r.eigenda != nil {
-		r.log.Debug("Storing data to eigenda backend")
-		return r.eigenda.Put(ctx, value)
+		r.log.Info("Storing data to eigenda backend")
+		//blob's commitment is verified and returned
+		result, err := r.eigenda.Put(ctx, value)
+		if err == nil {
+			if r.s3 != nil && r.s3.cfg.Backup {
+				r.log.Info("Storing data to S3 backend with key", "key", crypto.Keccak256(result))
+				ctx2, cancel := context.WithTimeout(ctx, time.Minute)
```

Comment:
it might make sense to have timeout duration as a user set s3 config value versus a hardset constant